src/bikeshed/core.clj

```diff
@@ -97,7 +98,7 @@
   [all-dirs & {:keys [max-line-length] :or {max-line-length 80}}]
   (printf "\nChecking for lines longer than %s characters.\n" max-line-length)
   (let [max-line-length (inc max-line-length)
-        cmd (str "find " all-dirs " -name "
+        cmd (str "find " all-dirs " find -regextype posix-extended -regex "
```

Comment:
These are incorrect, the command ends up being `find <dirs> find -regextype posixextended -regex ...`, note the second (extra) `find`